Sources/MessagingInApp/Gist/Managers/Models/Message.swift

```diff
@@ -18,9 +18,9 @@ public class GistProperties {
 
 public class Message {
     public private(set) var instanceId = UUID().uuidString.lowercased()
-    public let queueId: String?
+    public let queueId: String? // uniquely identifies an in-app message in the backend system
     public let priority: Int?
-    public let messageId: String
+    public let messageId: String // the messageId refers to the template used to render. For non-HTML messages, that messageId is something like "welcome-demo"
```

Comment:
if it is not big change, let's rename this to `templateId` as a messageId referencing a template isn't very conveying to what it actually is.

Reply:
I am fine with not doing this change in this PR for sure. Let's just not lose track of it as these hard to reason about names can easily fire back later.